app/views/reviews/new.html.erb

```diff
@@ -1,14 +1,9 @@
 <center>
-  <%= form_tag "/shelters/#{@shelter.id}", method: :post do %>
-    <p><%= label_tag :title %>
-    <%= text_field_tag :title %>
-    <p><%= label_tag :rating %>
-    <%= number_field_tag :rating %></p>
-    <p><%= label_tag :content %>
-    <%= text_field_tag :content %></p>
-    <p><%= label_tag :picture %>
-    <%= text_field_tag :picture %></p>
-
-  <%= submit_tag 'Submit' %>
-  <% end %>
-</center>
\ No newline at end of file
+  <%= render partial: 'form', locals: { path: "/shelters/#{@shelter.id}",
+  method: :post,
+  title: nil,
+  rating: nil,
+  content: nil,
+  picture: "https://www.moneyunder30.com/wp-content/uploads/2017/09/5_the-true-cost-of-pet-ownership-648x364-c-default.jpg",
```

Comment:
This is how i set the default value. We can take this out if you don't want to do it this way. It is kind of a run about way to do it.

Reply:
does it work?